com/course/zenkouv/src/Lecture3/Task1/resolver/WeaponResolver.java

```diff
@@ -18,4 +18,9 @@ public boolean resolve(List<Outfit> outfits, Outfit outfit) {
         }
         return true;
     }
+
+    @Override
+    public Weapon resolve(Outfit outfit) {
+        return (Weapon) outfit;
```

Comment:
you already knew what type of Outfit you have in list . remove this resolver method

Reply:
fix